modules/customizing-the-login-page.adoc

```diff
@@ -28,11 +27,6 @@ $ oc adm create-login-template > login.html
 ----
 $ oc adm create-provider-selection-template > providers.html
 ----
-+
-[source,terminal]
-----
-$ oc adm create-error-template > errors.html
-----
```

Comment:
why the removal of error-template-related commands?

Reply:
I may be wrong, but I don't see error page template renderers being used in the login page https://github.com/openshift/oauth-server/blob/822478f88514a80f053d9f65cd689d981b6ea4fd/pkg/oauthserver/auth.go#L354-L355 :thinking:

Reply:
as discussed OOB: templating still gets applied to the login page as it is being used by external IdPs to redirect to :+1: I suggest we simply enhance the description similar to how we did for the oc subcommand to create the error template.